cves/kernel/CVE-2020-15780.yml

```diff
@@ -151,10 +156,15 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: |
+    This bug was originally discovered by Jaons Donenfeld. They discovered it by 
```

Comment:
Possibly change who discovered the bug to their role rather than their name in the interest of anonymity, but might not be necessary